specs/deposits.md

```diff
@@ -296,6 +296,16 @@ Total calldata length MUST be exactly 164 bytes, implying the sixth and final se
 partially filled. This helps to slow database growth as every L2 block includes a L1 Attributes
 deposit transaction.
 
+In the first L2 block after the Ecotone activation block, the Ecotone L1 attributes are first used.
+
+The pre-Ecotone values are migrated over 1:1. The new Ecotone attributes are set as following:
+
+- The `baseFeeScalar` is set to the pre-Ecotone `scalar` value.
+- The `blobBaseFeeScalar` is set to 0.
+- The pre-Ecotone `overhead` attribute is dropped.
```

Comment:
How does this interact with the previous statement that they are migrated over 1:1?

Reply:
It's no longer used. I don't believe we necessarily need to specify the behavior of the overhead here. It just needs to be clear that it's no longer used. It being dropped is an implementation detail

Reply:
Right, but it lives in a bulleted section titled "The new Ecotone attributes are set as following". This isn't a new Ecotone attribute, right?

Reply:
I'm keen to keep it as-is, at least it's explicit that `overhead` is no longer used. It's a bit odd with the 1:1 migrated comment, but I don't have ideas for clearer wording.